### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by uarlouski**
- Remove unused GENERIC_CAPABILITIES variable from WebDriverFactory
- Add ability to set nested desired capabilities by separating levels of nesting using commas

### CODE REVIEW

File: vividus-util/src/main/java/org/vividus/util/property/PropertyParser.java [hunk 1]
```diff
@@ -43,6 +45,43 @@
...
+    {
+        Map<String, Object> container = new HashMap<>();
+        getPropertyValuesByPrefix(propertyPrefix).forEach((k, v) -> putByPath(container, k, v));
+        return container;
+    }
+
+    @SuppressWarnings("unchecked")
+    private static void putByPath(Map<String, Object> container, String path, String value)
+    {
+        String[] paths = path.split("\\.");
```
Comment (valfirst) [inline vividus-util/src/main/java/org/vividus/util/property/PropertyParser.java]: `org.apache.commons.lang3.StringUtils#split(java.lang.String)`

Comment (uarlouski) [reply vividus-util/src/main/java/org/vividus/util/property/PropertyParser.java]: changed to org.apache.commons.lang3.StringUtils#split(java.lang.String,char) ( space is default sep)

File: vividus-util/src/main/java/org/vividus/util/property/PropertyParser.java [hunk 2]
```diff
@@ -43,6 +47,43 @@
...
+                .limit(limit)
+                .reduce(container, (map, pathKey) ->
+                {
+                    Map<String, Object> nested = (Map<String, Object>) map.compute(pathKey, (k, v) ->
+                    {
+                        if (v == null)
+                        {
+                            return new HashMap<>();
+                        }
+                        else if (!(v instanceof Map))
```
Comment (ikalinin1) [inline vividus-util/src/main/java/org/vividus/util/property/PropertyParser.java]:
```java
else if (v instanceof Map)
{
    return v;
}
throw new IllegalArgumentException(String.format(
                                    "Path key '%s' from path '%s' is already used as a property key", pathKey, path));
```

**Comment by valfirst** [comment]
All changes have been cherry-picked to `master` branch

**Pull request closed without merging**